scripts/algorithm_evaluation_fixtures.py

```diff
@@ -53,6 +59,10 @@ def _get_inputs():
     )
 
 
+def _get_non_image_inputs():
+    return ComponentInterface.objects.filter(slug__in=["predictions-csv-file"])
+
```

Comment:
Don't we want a test with json type interface that is not stored in the DB as well?